pasqal_cloud/batch.py

```diff
@@ -112,6 +127,7 @@ def add_job(
             job_data["variables"] = variables
         job_rsp = self._client._send_job(job_data)
         job = Job(**job_rsp, _client=self._client)
+        self.ordered_jobs.append(job)
```

Comment:
I guess this is okay, as we add jobs, it will be created after the others, so it will be at the end of the list